src/app/inventory/store/stats.ts

```diff
@@ -505,10 +515,12 @@ export function interpolateStatValue(value: number, statDisplay: DestinyStatDisp
   const interp = statDisplay.displayInterpolation;
 
   // Clamp the value to prevent overfilling
-  value = Math.max(0, Math.min(value, statDisplay.maximumValue));
+  value = Math.min(value, statDisplay.maximumValue);
 
   let endIndex = interp.findIndex((p) => p.value > value);
-  if (endIndex < 0) {
+
+  // value < 0 is for mods with negative stats
+  if (endIndex < 0 || value < 0) {
```

Comment:
I'm not sure this is correct - if value < 0, shouldn't we use the first interpolation table entry, not the last?

Reply:
No because it would 0 cap them and I need that value to remain negative. I believe the table entires for all these stats are 0 and 42. This does line up with what bungie shows for those stats on items, but the mods themselves have negative values, which implies those table values done get used.